assets/control-plane/prometheus-rule.yaml

```diff
@@ -436,9 +436,9 @@ spec:
           }}' is experiencing {{ $value | humanizePercentage }} errors.'
         summary: Kubernetes API server client is experiencing errors.
       expr: |
-        (sum(rate(rest_client_requests_total{code=~"5.."}[5m])) by (cluster, instance, job, namespace)
+        (sum(rate(rest_client_requests_total{job="apiserver",code=~"5.."}[5m])) by (cluster, instance, job, namespace)
```

Comment:
maybe there are metrics for the OpenShift API server?

Reply:
![image](https://user-images.githubusercontent.com/2587585/235922909-80f25b0e-e2a2-47e4-8cb0-5eaa0edaf80b.png)